### DISCUSSION TITLE
Numpy y matplotlib

### DISCUSSION THREAD

**Discussion by MilcaGalvan** [LaunchX-InnovaccionVirtual/CursoIntroPython#36] [Q&A] [answered]
Hola buenas noches, alguien podría ayudarme? cuando estaba instalando numpy y manplotlib me salió un error y me quede atorada, les agradeceria mucho si me ayudaran 
![WhatsApp Image 2022-02-08 at 10 28 21 PM](https://user-images.githubusercontent.com/99236893/153124130-f54d89cc-ec46-41ce-a7d0-f64b21bd6c1a.jpeg)

**Comment by oscarbuao** [comment, marked as answer]
hola!! pude solucionarlo, actualiza los dos codigos tanto numpy como mat. 
ingresa este código: 
pip install --upgrade numpy
actualiza primero numpy y luego reinicia, luego cambia numpy por matplotlib
deja que termine de actualizar y luego reinicia.
después de actualizar los dos ya ingresa el comando que nos dan en la guía.

**Reply by JoseBenin82** [reply]
Amigo me di cuenta de algo en ello mas que los codigos , no copies de la pagina web y los pegues al editor del programa pues debes tu escrbir el propio codigo, al igual no se te olvide por cada instalacion de paquete o librerria  picarle en   Restart. Te  menciono todo ello  debido a que pase por la misma situacion jeeej, creeme que algo tan simple a veces nos complica la vida.

**Reply by JoseBenin82** [reply]
al igual intenta con %pip install numpy

**Reply by MilcaGalvan** [reply]
Vale! muchas gracias lo voy a checar.

**Reply by MilcaGalvan** [reply]
Muchas gracias! si sirvió lo que me dijiste @oscarbuao

**Reply by robertomoo** [reply]
Yo pase por lo mismo, creo que era la versión de Python tenía la 3.10 y utilizándola me daba error, por lo que estoy utilizando la 3.9 y ya me funciona sin problemas

**Reply by EnriquePH77** [reply]
Disculpa, como haces ese downgrade de python 3.10 al 3.9

**Reply by EnriquePH77** [reply]
> Disculpa, como haces ese downgrade de python 3.10 al 3.9

Ah ya ví esa parte del live. Muchas gracias de todos modos.

**Reply by MaribelMendezM** [reply]
con actualizar las librerías debe bastar

**Reply by semilun4** [reply]
¿Cómo actualizan las librerías? :c

**Reply by NeriCervantes05** [reply]
Muchas gracias @oscarbuao , me ayudo mucho, tenia el mismo problema

**Reply by Gracesl** [reply]
> hola!! pude solucionarlo, actualiza los dos codigos tanto numpy como mat. ingresa este código: pip install --upgrade numpy actualiza primero numpy y luego reinicia, luego cambia numpy por matplotlib deja que termine de actualizar y luego reinicia. después de actualizar los dos ya ingresa el comando que nos dan en la guía.

si funciona tu solucion , gracias.
